packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.ts

```diff
@@ -14,25 +18,33 @@ import Resize from '../../directives/resize'
 import Touch, { TouchWrapper } from '../../directives/touch'
 
 // Utilities
-import { convertToUnit } from '../../util/helpers'
+import { convertToUnit, getSlot } from '../../util/helpers'
 import mixins from '../../util/mixins'
 
 // TYpes
-import { VNode } from 'vue/types/vnode'
+import { VNode, VNodeDirective } from 'vue'
 import { PropValidator } from 'vue/types/options'
 
-export default mixins(
+const baseMixins = mixins(
```

Comment:
Why is it required?

Reply:
This is the similar style we use in the rest of the framework.